platform/mk3/capabilities.h

```diff
@@ -0,0 +1,97 @@
+#ifndef CAPABILITIES_H_
+#define CAPABILITIES_H_
+#include "serial.h"
+#include "FreeRTOSConfig.h"
+
+//Hardware capabilities for RCP MK2
+#define TICK_RATE_HZ	configTICK_RATE_HZ
+#define MS_PER_TICK	1
+
+/* Support Flags */
+#define BLUETOOTH_SUPPORT	1
+#define CELLULAR_SUPPORT	1
+#define LUA_SUPPORT		1
+#define SDCARD_SUPPORT		1
+#define USB_SERIAL_SUPPORT	1
+#define VIRTUAL_CHANNEL_SUPPORT	1
+#define WIFI_SUPPORT		1
+
+/* Wifi Specific Info */
+#define WIFI_MAX_BAUD       921600
+#define WIFI_MAX_SAMPLE_RATE    50
+#define WIFI_ENABLED_DEFAULT    true
+
+/* Rx Max Message length */
+#define RX_MAX_MSG_LEN	768
```

Comment:
Should this be 1024 or higher?  Or did we reduce the WiFi msg frame down and I missed it?